src/lime/_internal/backend/html5/HTML5Window.hx

```diff
@@ -805,7 +805,10 @@ class HTML5Window
 
 				if (touch == primaryTouch)
 				{
-					parent.clickCount = event.detail;
+					// touches shouldn't raise clickCounts value
+					// but they also indicate the end of a click sequence 
+					// (you expect click count to reset after you stop clicking with the mouse)
+					parent.clickCount = 0;
```

Comment:
When only using one finger, browsers will dispatch both a touch event (sans click count) and a click event (with click count). Proof: [this demo](https://developer.mozilla.org/en-US/docs/Web/API/Element/click_event#result) works on your phone, as long as you use only one finger.

So this code is either doing nothing or overwriting useful data. I'd say touch events just shouldn't interact with `clickCount` at all.

Reply:
Interesting! Didn't know that. Let me fix that real quick